src/misc/NumberWithCommas.js

```diff
@@ -0,0 +1,5 @@
+export default function numberWithCommas(x) {
+    let text = String(x);
+    return text.length > 3 ? text.replace(/\B(?=(\d{3})+(?!\d))/g, ",") : text;
```

Comment:
Have you tried `toLocaleString`? The Number object has a method of `toLocaleString`. You can do this to format your numbers to have commas instead 😀